lib/manager/maven/extract.ts

```diff
@@ -41,6 +41,11 @@ interface MavenProp {
   packageFile: string;
 }
 
+interface MavenPackageFile extends PackageFile {
+  mavenProps?: Record<string, any>;
+  parent?: string;
```

Comment:
should be moved to `managerData` ?

Reply:
It's actually just temporary during the extract, so is removed later before a `PackageFile` is returned.